pages.de/common/nix-collect-garbage.md

```diff
@@ -0,0 +1,17 @@
+# nix-collect-garbage
+
+> Löschen von unbenutzten und unerreichbaren Nix-Speicherpfaden.
+> Generationen können mit `nix-env --list-generations` aufgelistet werden.
+> Mehr Informationen: <https://nixos.org/releases/nix/latest/manual/#sec-nix-collect-garbage>.
+
+- Löschen von allen Speicherpfaden, die von den aktuellen Generationen der einzelnen Profile nicht verwendet werden:
+
+`sudo nix-collect-garbage --delete-old`
+
+- Simulieren der Löschung alter Speicherpfade:
+
+`sudo nix-collect-garbage --delete-old --dry-run`
+
+- Löschen aller Speicherpfade, die älter als 30 Tage sind:
+
+`sudo nix-collect-garbage --delete-older-than {{30d}}`
```

Comment:
The following fix could be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/nix-collect-garbage.md) page is updated:

```suggestion
`sudo nix-collect-garbage --delete-older-than 30d`
```

- Placeholders can be omitted for certain values.
- Our style guide doesn't specify whether this fix should be applied or not.